src/AzureDevOps.DataIngestor.Sdk/Entities/VssCommitEntity.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace AzureDevOps.DataIngestor.Sdk.Entities
+{
+    [Table("VssCommit")]
+    public partial class VssCommitEntity : VssBaseEntity

```

Comment:
>partial [](http://example.com/codeflow?start=11&length=7)

We probably don't need this to be partial correct?

Reply:
Probably not, I kinda copy from the PR one.